tests/regress/sql/test_activetable_limit.sql

```diff
@@ -22,7 +22,11 @@ INSERT INTO a03 values(generate_series(0, 500));
 CREATE EXTENSION diskquota;
 -- we only read the current log file
 CREATE EXTERNAL WEB TABLE segment_logs(line text)
-    EXECUTE 'cat $GP_SEG_DATADIR/pg_log/$(ls -Art $GP_SEG_DATADIR/pg_log | tail -n 1)'
+    EXECUTE 'if [ -d "$GP_SEG_DATADIR/pg_log" ]; then 
+            cat $GP_SEG_DATADIR/pg_log/$(ls -Art $GP_SEG_DATADIR/pg_log | tail -n 1)
+        else
+            cat $GP_SEG_DATADIR/log/$(ls -Art $GP_SEG_DATADIR/log | tail -n 1)
+        fi'
```

Comment:
why not using here gp_toolkit.__gp_log_segment_ext and below gp_toolkit.__gp_log_master_ext and filter them like
```sql
join pg_stat_activity a on logsession = 'con'||sess_id where pid = pg_backend_pid() and logmessage like '%the number of active tables reached the limit%'
```

Reply:
Can you please make a Github suggestion out of this change?

Reply:
I tested this and came to a conclusion that receiving the last log message from each segment using SQL will require a more clunkier code than this way.